.github/workflows/test.yml

```
@@ -38,4 +41,7 @@ jobs:
         run: forge build
 
       - name: Run tests
-        run: forge test -vvv
\ No newline at end of file
+        run: forge test -vvv
+
+      - name: Run tests
```

Maybe we should rename this?
```suggestion
      - name: Run SMA tests
```

---

Oops yes, thank you!